librz/core/cmd_type.c

```diff
@@ -1010,15 +1076,15 @@ RZ_IPI int rz_cmd_type(void *data, const char *input) {
 			if (name && type) {
 				name++; // skip the '.'
 				if (rz_str_startswith(type, "struct")) {
-					rz_core_cmdf(core, "tsc %s", name);
+					print_struct_union_in_c_format(TDB, stdifstruct, rz_str_trim_head_ro(input + 2), true);
 				} else if (rz_str_startswith(type, "union")) {
-					rz_core_cmdf(core, "tuc %s", name);
+					print_struct_union_in_c_format(TDB, stdifunion, rz_str_trim_head_ro(input + 2), true);
 				} else if (rz_str_startswith(type, "enum")) {
-					rz_core_cmdf(core, "tec %s", name);
+					print_enum_in_c_format(TDB, rz_str_trim_head_ro(input + 2), true);
 				} else if (rz_str_startswith(type, "typedef")) {
-					rz_core_cmdf(core, "ttc %s", name);
+					rz_core_list_typename_alias_c(core, input, TDB);
```

Comment:
same

Reply:
Missed this. Will fix in a later commit.